Astro Release Notes June 16, 2022 (#814)
astronomer/docs: astro/release-notes.md

**Latest CLI Version**: 1.0.1 ([Release notes](cli/release-notes.md))

## June 16, 2022

### Submit Support Requests in the Cloud UI

>  Support requests can now be created and submitted in the Cloud UI. You no longer need to register your help desk applications to submit Astronomer support requests. To simplify and speed the submission of support requests, the Support Request form is auto-populated with Workspace and Deployment selections for your organization.

Reviewer: Modify `You no longer need to register your help desk applications` to `You no longer need to register to [Astronomer Support](support.astronomer.io)...`

Author: @chomipi88 I have edited the text as we discussed on Slack. Thanks for the feedback!

>  Support requests can now be created and submitted in the Cloud UI. You no longer need to open an account on the Astronomer support portal to submit a support request. To simplify and speed the submission of support requests, the Support Request form is auto-populated with Workspace and Deployment selections for your organization.

Reviewer 2: ```suggestion
Support requests can now be created and submitted in the Cloud UI. You no longer need to open an account on the Astronomer support portal to submit a support request. To simplify and speed the submission of support requests, the Support Request form is auto-populated with Workspace and Deployment selections for your organization.
```

> To access the Support Request form in the Cloud UI, click the **Help** icon in the menu bar. 

Reviewer 2: ```suggestion
To access the Support Request form in the Cloud UI, see [Submit a Support Request in the Cloud UI](astro-support.md#submit-a-support-request-in-the-cloud-ui).
```

Reviewer 2: I know this link isn't active yet, but I assume you'll merge this with https://github.com/astronomer/docs/pull/822/ at the same time.

> 

Reviewer 2: ```suggestion

### Bug Fixes

- Fixed a rare issue where some user emails would be associated with the wrong username.
- Fixed an issue where you could not properly sort entries in the **People** tab by name.

```
These are the only documentable changes I found in the release log - there should be one more once Paola gets back to me on the node autoscaling/ parallelism feature.

**Latest CLI Version**: 1.0.1 ([Release notes](cli/release-notes.md))

## June 16, 2022

### Submit Support Requests in the Cloud UI

Support requests can now be created and submitted in the Cloud UI. You no longer need to open an account on the Astronomer support portal to submit a support request. To simplify and speed the submission of support requests, the Support Request form is auto-populated with Workspace and Deployment selections for your organization.

To access the Support Request form in the Cloud UI, see [Submit a Support Request in the Cloud UI](astro-support.md#submit-a-support-request-in-the-cloud-ui).

> 

Reviewer 2: ```suggestion

### A Deployment's Parallelism Now Autoscales with Worker Count

Deployments now autoscale Airflow's [`parallelism`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#parallelism) setting, which determines maximum number of tasks that can run concurrently within a single Airflow environment.

A Deployment's `parallelism` is now equal to the current number of workers multiplied by the value of [`worker_concurrency`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#worker-concurrency). This ensures that you won't be limited by a static parallelism limit as workers autoscale in your Deployment. 
```

**Latest CLI Version**: 1.0.1 ([Release notes](cli/release-notes.md))

## June 16, 2022

### Submit Support Requests in the Cloud UI

Support requests can now be created and submitted in the Cloud UI. You no longer need to open an account on the Astronomer support portal to submit a support request. To simplify and speed the submission of support requests, the Support Request form is auto-populated with Workspace and Deployment selections for your organization.

To access the Support Request form in the Cloud UI, see [Submit a Support Request in the Cloud UI](astro-support.md#submit-a-support-request-in-the-cloud-ui).

### A Deployment's Parallelism Now Autoscales with Worker Count

Deployments now autoscale Airflow's [`parallelism`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#parallelism) setting, which determines maximum number of tasks that can run concurrently within a single Airflow environment.

> A Deployment's `parallelism` is now equal to the current number of workers multiplied by the value of [`worker_concurrency`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#worker-concurrency). This ensures that you won't be limited by a static parallelism limit as workers autoscale in your Deployment. 

Reviewer 2: ```suggestion
A Deployment's `parallelism` is now equal to the current number of workers multiplied by the value of [`worker_concurrency`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#worker-concurrency). This ensures that you won't be limited by a static parallelism limit as workers autoscale in your Deployment. 

Note that you can still use a static `parallelism` value by setting `AIRFLOW__CORE__PARALLELISM` as an [environment variable](environment-variables.md).
```

> A Deployment's `parallelism` is now equal to the current number of workers multiplied by the [`worker_concurrency`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#worker-concurrency) value. This change ensures that you won't be limited by a static parallelism limit as workers autoscale in your Deployment. 

Reviewer 2: ```suggestion
A Deployment's `parallelism` is now equal to the current number of workers multiplied by the [`worker_concurrency`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#worker-concurrency) value. This change ensures that you won't be limited by a static parallelism limit as workers autoscale in your Deployment. See [Worker Autoscaling Logic](configure-deployment-resources.md#worker-autoscaling-logic) for more information. 

Note that you can still use a static `parallelism` value by setting `AIRFLOW__CORE__PARALLELISM` as an [environment variable](environment-variables.md).
```

To access the Support Request form in the Cloud UI, see [Submit a Support Request in the Cloud UI](astro-support.md#submit-a-support-request-in-the-cloud-ui).

### A Deployment's Parallelism Now Autoscales with Worker Count

Deployments now autoscale to the Airflow [`parallelism`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#parallelism) setting, which determines the maximum number of tasks that can run concurrently within a single Airflow environment.

A Deployment's `parallelism` is now equal to the current number of workers multiplied by the [`worker_concurrency`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#worker-concurrency) value. This change ensures that you won't be limited by a static parallelism limit as workers autoscale in your Deployment. See [Worker Autoscaling Logic](configure-deployment-resources.md#worker-autoscaling-logic) for more information. 

> Note that you can still use a static `parallelism` value by setting `AIRFLOW__CORE__PARALLELISM` as an [environment variable](environment-variables.md).

Reviewer 2: ```suggestion
Note that you can still use a static `parallelism` value by setting `AIRFLOW__CORE__PARALLELISM` as an [environment variable](environment-variables.md).

```

**Latest CLI Version**: 1.1.0 ([Release notes](cli/release-notes.md))

## June 16, 2022

### Submit Support Requests in the Cloud UI

> Support requests can now be created and submitted in the Cloud UI. You no longer need to open an account on the Astronomer support portal to submit a support request. To simplify and speed the submission of support requests, the Support Request form is auto-populated with Workspace and Deployment selections for your organization.

Reviewer 2: Tidied this up a bit

To access the Support Request form in the Cloud UI, see [Submit a Support Request in the Cloud UI](astro-support.md#submit-a-support-request-in-the-cloud-ui).

### A Deployment's Parallelism Now Autoscales with Worker Count

> Deployments now autoscale to the Airflow [`parallelism`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#parallelism) setting, which determines the maximum number of tasks that can run concurrently within a single Airflow environment.

Reviewer 3: ```suggestion
In Airflow, there are multiple settings that ultimately control how many tasks can run concurrently for a given Deployment, worker, or DAG. To enable seamless task growth, Astro now dynamically calculates [`parallelism`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#parallelism), which is an Airflow configuration that determines the maximum number of tasks that can run concurrently within a single Deployment.
```

> A Deployment's `parallelism` is now equal to the current number of workers multiplied by the [`worker_concurrency`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#worker-concurrency) value. This change ensures that you won't be limited by a static parallelism limit as workers autoscale in your Deployment. See [Worker Autoscaling Logic](configure-deployment-resources.md#worker-autoscaling-logic) for more information. 

Reviewer 3: ```suggestion
Instead of being set to 32 by default, `parallelism` for each Deployment is now set to a number that is equal to the current number of running workers multiplied by [`worker_concurrency`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#worker-concurrency). This change ensures that the ability for your Deployment to autoscale isn't artificially limited by a maximum that is not visible to most users. See [Worker Autoscaling Logic](configure-deployment-resources.md#worker-autoscaling-logic) for more information. 
```

WDYT?

Reviewer 2: Seems a bit like we're laboring on about how bad it was before, I think the benefit of the change is straightforward enough that we can focus mostly on how it is today vs. how it was.

> Note that you can still use a static `parallelism` value by setting `AIRFLOW__CORE__PARALLELISM` as an [environment variable](environment-variables.md).

Reviewer 3: ```suggestion
For users with advanced use cases, note that you can override this new default by setting `AIRFLOW__CORE__PARALLELISM` as an [environment variable](environment-variables.md).
```

Reviewer 2: If we can't claim an advanced use case here, I don't think we should promise this.
